ibraupdate.sh

```diff
@@ -2,14 +2,22 @@
 
 ######################################################################
 # Title   : Beachhead - Initial Installer for IBRAMENU
-# By      : DiscDuck
+# By      : DiscDuck, Taos15
 # License : General Public License GPL-3.0-or-later
 # Another fine product brought to you by IBRACORP™
 ######################################################################
 
 sudo bash -c "rm -R /opt/ibracorp/ibramenu/"
-sudo bash -c "git clone -b traefik2 --single-branch https://github.com/taos15/ibramenu.git /opt/ibracorp/ibramenu"
-cd find $ifolder -type f -iname "*.sh" -exec chmod +x {} \;
+sudo bash -c "git clone -b main --single-branch https://github.com/taos15/ibramenu.git /opt/ibracorp/ibramenu"
+sudo find $ifolder -type f -iname "*.sh" -exec chmod +x {} \;
 
+# update the custom docker netwrok use in all the containers
+update_docker_network () {
+  read -p "Enter the name of your custome docker network (ex. ibranet) : " customnetwork
+  echo "dockernet=$customnetwork" >> /opt/ibracorp/ibramenu/.profile
+  docker network create $customnetwork > /dev/null 2>&1
+}
```

Comment:
Now when we install docker we can choose the custom docker network’s name. It also ask every time we use ibraupdate